dartsim/src/SimulationFeatures.hh

```diff
@@ -39,7 +39,7 @@ struct SimulationFeatureList : FeatureList<
 > { };
 
 class SimulationFeatures :
-    public CanWriteRequiredData<SimulationFeatures, ForwardStep::Output>,
+    public CanWriteExpectedData<SimulationFeatures, ForwardStep::Output>,
```

Comment:
This can be limited to just ` ExpectData<ChangedWorldPoses>` and then we don't have to create empty implementations.

```suggestion
    public CanWriteExpectedData<SimulationFeatures, ExpectData<ChangedWorldPoses>>,
```

Reply:
Thanks for the tip, I used this in https://github.com/ignitionrobotics/ign-physics/pull/238/commits/1433ddbc73999fc149e25dc206448f4ee7b52ac2 and the empty implementations should no longer be needed.